website/docs/developer/development-guide/code/best-practices.mdx

```diff
@@ -27,43 +27,42 @@ except requests_db.DatabaseError as err:
 if not success:
     logging.error(f"You may log additional information if desired. "
                   f"param: {param}")
-    raise DescriptiveErrorType(f'Error message to be raised info Cumulus workflow.'
+    raise DescriptiveErrorType(f'Error message to be raised info Cumulus workflow.')
 ```
 Retries can then be configured in the workflow json if desired. See
 [documentation](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-error-handling.html)
 and
 [tutorials](https://aws.amazon.com/getting-started/hands-on/handle-serverless-application-errors-step-functions-lambda/)
 for more information.
+The following snippet from the copy_to_glacier lambda demonstrates usage of retries for a lambda in an ingest workflow.
+`MaxAttempts` is set to 6, meaning that it will run the function a maximum of 7 times before transitioning to the `WorkflowFailed` state.
+`IntervalSeconds` determines how many seconds the workflow will sleep between retries.
+A `BackOffRate` of 2 means that the `IntervalSeconds` will be doubled on each failure beyond the first.
+
 ```json
-"SomeState": {
+"CopyToGlacier": {
   ...
+  "Type": "Task",
+  "Resource": "${copy_to_glacier_task_arn}",
   "Retry": [
     {
       "ErrorEquals": [
-        "Lambda.DescriptiveErrorType",
-        "Lambda.AnotherErrorType"
+        "Lambda.ServiceException",
+        "Lambda.AWSLambdaException",
+        "Lambda.SdkClientException"
       ],
       "IntervalSeconds": 2,
       "MaxAttempts": 6,
       "BackoffRate": 2
     }
   ],
-  "Catch": [
-    {
-      "ErrorEquals": [
-        "States.ALL"
-      ],
-      "ResultPath": "$.exception",
-      "Next": "WorkflowFailed"
-    }
-  ],
-  "Next": "NextState"
-}
+  "Next": "WorkflowSucceeded"
```

Comment:
I forgot to add the catch part in the example.

You would want to add that back in.

```json
  "Catch": [
    {
      "ErrorEquals": [
        "States.ALL"
      ],
      "ResultPath": "$.exception",
      "Next": "WorkflowFailed"
    {
  ],
```